packages/office-ui-fabric-react/src/utilities/ButtonGrid/ButtonGrid.types.ts

```diff
@@ -23,7 +23,7 @@ export interface IButtonGridProps extends React.TableHTMLAttributes<HTMLTableEle
   /**
    * Custom renderer for the individual items.
    */
-  onRenderItem: (item: any, index: number) => JSX.Element;
+  onRenderItem: (item: any, index: number) => JSX.Element | null;
```

Comment:
Can you please explain why change in return type is required?

Reply:
The onRenderItem function invokes onRenderColorCell which is of IRenderFunction type and it return JSX.Element | null. 
Not changing the return type will make compiler complain.